doc/nrf/links.txt

```diff
@@ -219,6 +219,10 @@
 
 .. _`BH1749NUC-E`: https://www.rohm.com/datasheet/BH1749NUC/bh1749nuc-e
 
+.. _`JSON`: https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/working-with-json.html
+
+.. _`Setting up an AWS IoT account`: https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/
```

Comment:
```suggestion
.. _`Setting up an AWS account`: https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/
```

Linked guide is about setting up an AWS account generally, not specific to AWS IoT.